jest.config.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
     global: {
       statements: 80,
       branches: 73,
-      functions: 80,
+      functions: 78,
```

Comment:
Can we add additional tests for this component to meet the threshold instead of decreasing the threshold?

Reply:
Scott said to just push it for now